scss/list.scss

```diff
@@ -4,7 +4,7 @@
 .cards-container {
   width: 100%;
   max-width: 900px;
-  margin: 50px auto;
+  margin: 104px auto;
```

Comment:
it would be a good idea to store the height of the navbar and use it to calculate this margin. If someday the height of the navabar changes you will have to update this margin too